src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/ConnectionPoint/IEventSource.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Runtime.InteropServices;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.ConnectionPoint
+{
+    /// <summary>
+    /// This implementation is a copy from CPS
+    /// </summary>
+    [ComVisible(false)]
```

Comment:
We dont want `IEventSource` to be Com Visible which was the original intent. Why should one remove it?